### PR TITLE
263. Ugly Number

### PR DESCRIPTION
**Pull request by Admarx**

### CODE REVIEW

**Review by MateuszSapala — DISMISSED** [review]

**Review by Wegrzynk — APPROVED** [review]

**Review by ElizaWielocha — APPROVED** [review]

**Review by barbara-k97 — DISMISSED** [review]

File: UglyNumber/UglyNumber/UglyNumber.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            else
+                Console.WriteLine(num + " is NOT an ugly number");
+
+            if (IsUgly(num2))
+                Console.WriteLine(num2 + " is an ugly number");
+            else
+                Console.WriteLine(num2 + " is NOT an ugly number");
+        }
+    }
+}
```
Comment (ArsenEloglian) [inline UglyNumber/UglyNumber/UglyNumber.cs:55]: Please add endline

File: UglyNumber/UglyNumber/UglyNumber.csproj [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (ArsenEloglian) [inline UglyNumber/UglyNumber/UglyNumber.csproj:1]: These file changes should be ignored by .gitignore
Please update gitignore file.
Please clean up your folder.

**Review by ArsenEloglian — CHANGES_REQUESTED** [review]

File: UglyNumber/UglyNumber/UglyNumber.cs (hunk 1 above)
Comment (Admarx) [reply UglyNumber/UglyNumber/UglyNumber.cs]: Endlines have been added

File: UglyNumber/UglyNumber/UglyNumber.csproj (hunk 2 above)
Comment (Admarx) [reply UglyNumber/UglyNumber/UglyNumber.csproj:1]: I'll make a separate pull request cleaning up everything, and then one modifying the gitignore.

**Review by barbara-k97 — APPROVED** [review]

**Comment by Admarx** [comment]
@ArsenEloglian Could you review again? I commented in the conversations before resolving them.

**Review by MateuszSapala — APPROVED** [review]

File: UglyNumber/UglyNumber/UglyNumber.csproj (hunk 2 above)
Comment (ArsenEloglian) [reply UglyNumber/UglyNumber/UglyNumber.csproj:1]: o.k

**Review by ArsenEloglian — APPROVED** [review]

**Pull request merged**